ai_economist/foundation/env_wrapper.py

```diff
@@ -14,7 +14,11 @@
     num_gpus_available = len(GPUtil.getAvailable())
     assert num_gpus_available > 0, "This script needs a GPU to run!"
```

Comment:
I should have brought this up in an earlier PR, but I think we may want to avoid organizing GPU-required files as top-level files within `ai_economist/foundation/`.

As a suggestion for future PRs, I recommend some minor refactoring. If `env_wrapper.py` has a specific use case (i.e training with WarpDrive), that should be reflected somehow.

Reply:
Thanks for the suggestion. Made the env wrapper CPU-friendly too.